docs/understanding-docker.md

```diff
@@ -209,25 +199,24 @@ daemon to run a container.
 
     $ docker run -i -t ubuntu /bin/bash
 
-Let's break down this command. The Docker client is launched using the `docker`
-binary with the `run` option telling it to launch a new container. The bare
-minimum the Docker client needs to tell the Docker daemon to run the container
-is:
+The Docker client is launched using the `docker` binary with the `run` option
```

Comment:
Here are our current terminology guidelines:

Docker Engine is the core product which provides image and container functionality.  

First use on a page within text, not menu titles.

- Docker Engine 
- Docker Engine daemon 
- Docker Engine CLI
- a Docker Engine client 
- Docker Engine host
- Docker Engine Remote API

Subsequent references in text

- Engine daemon
- Engine CLI
- Engine CLI commands
- an Engine command
- an Engine client
- Engine Remote API
- Engine host
- Engine